lib/Styled.ts

```diff
@@ -1,28 +1,14 @@
 import styled, { ThemeProps } from 'styled-components';
 import { Elements } from './Attributes';
 
-export interface ComponentTheme {}
-
-export interface DivTheme extends ComponentTheme {
-    colorTextRest: string;
-    colorBackground: string;
-}
-
-export interface ButtonTheme extends ComponentTheme {
-    colorRest: string;
-    colorHover: string;
-    colorDisabled: string;
-    colorTextRest: string;
+export interface ButtonTheme {
+    colorRest?: string;
```

Comment:
I think we should add Bg in here like 'colorBgRest' to align with the existing name pattern, what do you think?

Reply:
agree